modules/EvseManager/Charger.cpp

```diff
@@ -998,7 +998,7 @@ void Charger::update_pwm_now(float dc) {
     session_log.evse(
         false,
         fmt::format(
-            "Set PWM On ({}%) took {} ms", dc * 100.,
+            "Set PWM On ({.1f}%) took {} ms", dc * 100.,
```

Comment:
```suggestion
            "Set PWM On ({:.1f}%) took {} ms", dc * 100.,
```

Otherwise EvseManager crashes with:
`terminate called after throwing an instance of 'fmt::v10::format_error'
  what():  invalid format string`